content/docs/rfcs/37/README.md

```diff
@@ -0,0 +1,151 @@
+This effectively emulates a mechanism to _"re-activate"_ a stale Noise session by binding it to a newly created active Noise session.
+
+In order to reduce users' metadata leakage, it is desirable (as suggested in [35/WAKU2-NOISE](https://rfc.vac.dev/spec/35/#after-handshake)) that content topics used for communications change every time a new message is exchanged.
+This can be easily realized by employing a key derivation function to compute a new `session-id` from the previously employed one (e.g. `session-id = HKDF(prev-session-id)`),
+while keeping the Inbound/outbound Cipher States, the content topic derivation mechanism and the stale mechanism the same as above.
+In this case, when one party sends **and** receives at least one message, 
+he SHALL publicly mark as stale all Noise sessions relative to messages exchanged before the earlier of these two send/receive events.
+
+# Multi-Device support
+
+Alice and Bob might possess one or more devices (e.g. laptops, smartphones, etc.) they wish to use to communicate.
+In the following we assume Alice and Bob to possess $N$ and $M$ devices, respectively.
+
+Since a Noise session contains cryptographic material required to encrypt and decrypt messages exchanged on a pre-defined content topic derived from a `session-id`, 
+messages should be encrypted and decrypted within the Noise session instantiated between the currently-in-use sender's and receiver's device.
+
+This is achieved through two main supported session management mechanisms that we called `N11M` and `NM`, respectively.
+
+## The $N11M$ session management mechanism
+
+In a $N11M$ setting, each party's device shares the same Noise session information used to encrypt and decrypt messages exchanged with the other party.
+
+![](./N11M.png)
+
+More precisely, once the first Noise session between any of Alice's and Bob's device is instantiated, 
+its session information is securely propagated to all other devices, 
+which then become able to send and receive new messages on the content topic associated to such session.
+
+The most updated session information between Alice and Bob is propagated in encrypted form to other devices, 
+using previously instantiated Noise sessions. 
+The propagated session information corresponds to the latest session information stored on the device currently communicating with (any of the devices of) the other party.
+
+We note that sessions information is propagated only among devices belonging to the same party and not with other party's devices. 
+Hence, Alice has no knowledge on the number of devices Bob is using and vice versa. 
+
+When any device marks a Noise session between Alice and Bob as stale, 
+all (updated) devices will implicitly mark such session as stale.
```

Comment:
```suggestion
all (updated) devices will mark such a session as stale upon synchronization with the active device.
```

Reply:
The stale is implicit here, which means that the session is marked stale with no previous/further action from other devices. Since the device that marked the session stale does it by publishing a message on the corresponding content topic, and all Alice's (updated) device will share the same session information and will read such message, devices will see that the session was marked stale (even if they didn't to directly) and no further action is required from them (besides application specific actions such as deleting session information etc.).

Reply:
The ambiguity is around the `implicit` term, Do you mean they `internally`/`'locally` mark it as stale?

Reply:
I see! The fact is that other devices don't mark it as stale. The marking is done online by one device, will others will just see the session to be marked as stale. So, since the session information is shared, this is equivalent to the fact that each device marked it as stale, but only one actually did it: all the others did it _implicitly_! I cannot think about another word to express this!

Reply:
I see your point, maybe `they consider the corresponding Noise session as stale and act accordingly`, though, no pressure!

Reply:
Addressed in https://github.com/vacp2p/rfc/pull/504/commits/9cff4db8f1ade87e9dada9cc7eb93f6cce55b13f